pkg/rhel/openvdc.spec

```diff
@@ -93,6 +94,12 @@ OpenVDC executor common package.
 /opt/axsh/openvdc/share/mesos-slave/attributes.lxc
 /opt/axsh/openvdc/share/lxc-templates/lxc-openvdc
 %dir /etc/openvdc
+%dir /etc/openvdc/ssh
+
+%post executor
+test ! -f /etc/openvdc/ssh/host_rsa_key && /usr/bin/ssh-keygen -q -t rsa -f /etc/openvdc/ssh/host_rsa_key -C '' -N '' >&/dev/null;
```

Comment:
Default of rsa key size is 2048bit, therefore I'll modify the size of rsa key from 2048 to 4096.